epsagon/events/botocore.py

```diff
@@ -117,13 +117,82 @@ def update_response(self, response):
         :param response: Response from botocore
         :return: None
         """
-        self.event_id = response['ResponseMetadata']['RequestId']
+        event_id = response['ResponseMetadata'].get('RequestId')
+        if event_id:
+            self.event_id = event_id
         self.resource['metadata']['Retry Attempts'] = \
             response['ResponseMetadata']['RetryAttempts']
         self.resource['metadata']['Status Code'] = \
             response['ResponseMetadata']['HTTPStatusCode']
 
 
+class BotocoreCloudWatchEvent(BotocoreEvent):
+    """
+    Represents cloudwatch events (eventbridge) botocore event.
+    """
+
+    RESOURCE_TYPE = 'eventbridge'
+    RESOURCE_TYPE_UPDATE = 'events'
+
+    def __init__(
+        self, wrapped, instance, args, kwargs, start_time, response, exception
+    ):
+        """
+        Initialize.
+        :param wrapped: wrapt's wrapped
+        :param instance: wrapt's instance
+        :param args: wrapt's args
+        :param kwargs: wrapt's kwargs
+        :param start_time: Start timestamp (epoch)
+        :param response: response data
+        :param exception: Exception (if happened)
+        """
+
+        super(BotocoreCloudWatchEvent, self).__init__(
+            wrapped, instance, args, kwargs, start_time, response, exception
+        )
+        _, request_data = args
+        entries = request_data.get('Entries', [{}])[0]
```

Comment:
I would change it so you initially get 'Entries' and then check it (so it won't explode if request_data.get('Entries') is empty)